### DISCUSSION TITLE
Cannot .gitignore browser-sync.config.js?

### DISCUSSION THREAD

**Discussion by wpbuilds** [understrap/understrap-child#351] [Q&A] [answered]
Hi there,
               i feel like I am missing something really obvious ..... but adding 'src/build/browser-sync.config.js' to my .gitignore (that is bundled with the child-theme) does not exclude it? What am i missing?

**Comment by bacoords** [comment, marked as answer]
If it's already committed - you'll need to delete it, then commit the gitignore change as a separate commit, I think.

**Reply by wpbuilds** [reply]
Ok, I'll give that a go and report back. Thanks @bacoords

**Reply by wpbuilds** [reply]
@bacoords tried your approach... no joy for me? 

Steps to reproduce:

1) Delete already committed .gitignore from my existing child-theme repo.
2) Update local by pulling from remote.
3) Recreate .gitignore file using default child-theme settings + include my new line: 'src/build/browser-sync.config.js'
4) Push to remote.
5) Edit my local 'browser-sync.config.js' with settings I do NOT want tracked.
--- Change is tracked?

Any Ideas? Or have I misunderstood your instruction @bacoords?

**Reply by bacoords** [reply]
Yeah sorry - you'll need to basically make a commit of you deleting the config.js file - git needs to see it deleted. Then you can add it to your gitignore, commit that, and then put the file back.

**Reply by wpbuilds** [reply]
Ahh got it. Appreciate your help 🙂
